### ISSUE TITLE
Add support for more recent versions of libicu

### ISSUE THREAD

**Issue by lloydwatkin** [astro/node-stringprep#62]
See http://icu-project.org/apiref/icu4c/

API changes are documented in there.  It seems the most recent versions of ubuntu are using 53, we should attempt to support the newer versions.

**Comment by sonnyp** [comment]
Hey, one of our system guy is telling me this:
```
Yeah. I've built it OK on 14.04 VM with libicu-dev 52.
Ubuntu 12.04 we use in prod does have old libicu-dev 48.
Both build OK with node 0.12. I have no warnings I had with 0.10.
```

**Comment by lloydwatkin** [comment]
Yeah I've had no issues on ubuntu with multiple version of libicu too.

**Comment by lloydwatkin** [comment]
This issue arises because people were having issues at reporting it to be version issues, but I think that was not actually the issue.

**Comment by sonnyp** [comment]
I guess we can just ask them to use Node.js 0.12 then.
